DemoApp/DemoAppCoordinator.swift

```diff
@@ -66,15 +66,23 @@ final class DemoAppCoordinator: NSObject, UNUserNotificationCenterDelegate {
 
     func presentChat(userCredentials: UserCredentials, channelID: ChannelId? = nil) {
         // Create a token
-        let token = try! Token(rawValue: userCredentials.token)
```

Comment:
This wasn't really scope of the ticket, but avoiding force unwraps in the DemoApp